src/views/Hero/Hero.scss

```diff
@@ -0,0 +1,11 @@
+.HeroView {
+  height: 100%;
+}
+
+.HeroView__homeView {
+  background: linear-gradient(180deg, var(--wedgewood__gradient) 0%, var(--waterleaf) 100%);
+  height: 90%;
+  border-bottom-left-radius: 50px;
+  border-bottom-right-radius: 50px;
```

Comment:
Also, are we keeping this for the landing page? (Or is the Hero different from the Landing?)

Reply:
From my understanding of what the final UI will look like, I think each view (Home/Landing, Schedule, etc) will have its own theme. So this landing postcard goes along with this gradient but I'm guessing the other views will have a separate gradient as well.

Reply:
Oh I see. I was mainly confused about the rounded bottom corners since I was looking at the Minimal panel instead of the Full Implementation panel on Figma. It makes sense now - thanks for the clarification.